### PR TITLE
TCVP-1645 updated OCR validation rules

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

TCVP-1645

OCR validation rule changes, test changes

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Requirements change
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

dotnet test

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/TrafficCourts/Citizen.Service/Validators/Rules/OnlyMVAIsSelectedRule.cs [hunk 1]
```diff
@@ -26,10 +26,12 @@ public override Task RunAsync()
         bool? tcr = _violationTicket.Fields[OcrViolationTicket.OffenceIsTCR].IsCheckboxSelected();
         bool? other = _violationTicket.Fields[OcrViolationTicket.OffenceIsOther].IsCheckboxSelected();
 
-        if (!mva ?? false)
-        {
```
Comment (lfdgov) [inline src/backend/TrafficCourts/Citizen.Service/Validators/Rules/OnlyMVAIsSelectedRule.cs:30]: remove commented out code

**Review by lfdgov — APPROVED** [review]

File: src/backend/TrafficCourts/Citizen.Service/Validators/Rules/OnlyMVAIsSelectedRule.cs (hunk 1 above)
Comment (KDNeufeld) [reply src/backend/TrafficCourts/Citizen.Service/Validators/Rules/OnlyMVAIsSelectedRule.cs:30]: done

File: src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountActRegMustBeMVA.cs [hunk 2]
```diff
@@ -3,12 +3,11 @@
 namespace TrafficCourts.Citizen.Service.Validators.Rules;
 
 /// <summary>
-/// For all 3 counts, the ACT/REGs section must be "MVA" or blank.
+/// For all 3 counts, the ACT/REGs section must be "MVA", "MVR", or blank.
```
Comment (karahanb) [inline src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountActRegMustBeMVA.cs:6]: As far as I understood from the requirements and acceptance criteria, ACT/REGs section must not be blank anymore. Must have "MVA" or "MVR" written in that box to count as MVA ticket. So should we update the if check below based on that to not allow blank box?

Comment (KDNeufeld) [reply src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountActRegMustBeMVA.cs:6]: I had the same question, but then I realized that not all 3 counts must be filled in - some tickets may only have 1 or 2 counts. In such cases this field will be blank on the blank count row.

Comment (karahanb) [reply src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountActRegMustBeMVA.cs:6]: Ah I see, that makes sense.

**Pull request merged**